mysql-test/suite/rocksdb/t/add_index_inplace.test

```diff
@@ -255,7 +255,7 @@ SET @prior_rocksdb_merge_buf_size = @@rocksdb_merge_buf_size;
 
 SET global rocksdb_strict_collation_check = off;
 SET session rocksdb_merge_combine_read_size = 566;
-SET session rocksdb_merge_buf_size = 336;
+SET session rocksdb_merge_buf_size = 340;
```

Comment:
Update: The bitmap is now used only for partially covering key parts and is not stored if there are no such key parts.